assignments/array-methods.js

```diff
@@ -56,28 +56,70 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function (object) {
+    let name = object.first_name + " " + object.last_name;
+    fullName.push(name)
+})
 console.log(fullName);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+const cap = function(first) {
+    return first.toUpperCase();
+}
+
+allCaps = fullName.map(cap)
 console.log(allCaps); 
 
```

Comment:
Could do this in one line with map function like this : 
`let allCaps = runners.map((person) => {
  return person.first_name.toUpperCase();
});
console.log(allCaps); 
`